nuitka/plugins/standard/standard.nuitka-package.config.yml

```diff
@@ -2494,7 +2511,7 @@
   implicit-imports:
     - depends:
         - 'numpy.core._dtype_ctypes'
-        - 'numpy.core._multiarray_tests'
+        - 'numpy._core._multiarray_tests'
```

Comment:
Never remove config needed for older versions, this should have added the change as a new one. Nuitka will ignore things not found. Also I have a suspect, that `numpy.core` configuration elsewhere might have to be adjusted as well.